File: cli/layout/default_layout.py
layout = {
    "version": 3,
    "layout": [
        "Weather for {@region}, {@country}",
        [
            {
                "type": "variable",
                "value": "current_weather.condition_sentence",
                "color": "LIGHTMAGENTA_EX",
            }
        ],
        [
            {
                "type": "variable",
                "value": "forecast_sentence",
                "color": "LIGHTMAGENTA_EX",
            }
        ],
        "Temperature: {@current_weather.temperature|° F|° C} with a low of {@current_weather.min_temp|° F|° C}, "
        "and a high of {@current_weather.max_temp|° F|° C}, feels like {@current_weather.feels_like|° F|° C}",
        "Wind: {@current_weather.wind.speed| mph| km/h} at {@current_weather.wind.heading|°}",
        "Cloud Cover: {@current_weather.cloud_cover|%}",
        "Dew Point: {@current_weather.dewpoint|° F|° C}",
        "AQI: {#color_aqi|@current_weather.aqi}",
    ],
}

File: dev-scripts/src/default_layout.rs
pub fn get_default_layout() -> String {
"NAME = DEFAULT
VERSION = 22
LAYOUT_VERSION = 5
--------------------------------
Weather for {@location.city}, {@location.state}, {@location.country}
{$FORE_LIGHTMAGENTA$@weather.condition_sentence}
{$FORE_LIGHTMAGENTA$@forecast_sentence}
{$BOLD$}Temperature{$RESET$}: {@weather.temperature|° F|° C} with a low of {@weather.min_temp|° F|° C}, and a high of {@weather.max_temp|° F|° C}, feels like {@weather.feels_like|° F|° C}
{$BOLD$}Wind{$RESET$}: {@weather.wind.speed| mph| km/h} at {@weather.wind.heading|°}
{$BOLD$}Cloud Cover{$RESET$}: {@weather.cloud_cover|%}
{$BOLD$}Dew Point{$RESET$}: {@weather.dewpoint|° F|° C}
{$BOLD$}AQI{$RESET$}: {#color_aqi|@weather.aqi}".to_string()
}
